src/migrations/20200119150631-create-likes.js

```diff
@@ -0,0 +1,36 @@
+export const up = (queryInterface, Sequelize) => queryInterface.createTable('likes', {
+  id: {
+    allowNull: false,
+    autoIncrement: true,
+    primaryKey: true,
+    type: Sequelize.INTEGER
+  },
+  accommodationId: {
+    type: Sequelize.INTEGER
+  },
+  userId: {
+    type: Sequelize.INTEGER
+  },
+  like: {
+    type: Sequelize.BOOLEAN,
+    defaultValue: false
+  },
+  unlike: {
+    type: Sequelize.BOOLEAN,
+    defaultValue: false
+  },
```

Comment:
There isn't a point saving `unlike`s. If a user clicks on the like button twice, remove it from the database instead. Only store `like`s.

Reply:
@adesege ,  On the UI we will need to know counts of `likes` and `unlikes`. That's why we added `unlike`s column to the table. Thank you